data/getAllRoundScores.js

```diff
@@ -0,0 +1,68 @@
+const { google } = require('googleapis');
```

Comment:
This file duplicates a lot of code from `generateData`. Could you factor it out?

Reply:
Yeah, totally duplicated. The plan was to see if you liked this approach, then convert stats.js to use `getAllRoundScores.js` and delete `generateData.js`